vendor-extra/JatsContentBundle/src/EventListener/ContentHeaderListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ContentPageBundle\Event\CreateContentPageEvent;
+use Libero\ViewsBundle\Views\ViewConverter;
+
+final class ContentHeaderListener
+{
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    public function onCreatePage(CreateContentPageEvent $event) : void
+    {
+        $xpath = $event->getItem()->xpath();
+
+        $front = $xpath->firstOf('/libero:item/jats:article/jats:front');
+
+        if (!$front instanceof Element) {
```

Comment:
should these checks be positive, e.g. `$front === null`?

Reply:
This helps static analysis (and maybe humans) understand the type, since XPath can return many different types.